signal2html/html.py

```diff
@@ -16,6 +16,7 @@
 from types import SimpleNamespace as ns
 
 from .html_colors import COLORMAP
```

Comment:
Is this import still needed somewhere else?

Reply:
Yes it's still used to cycle through group member colors to ensure they are unique ([line 171](https://github.com/GjjvdBurg/signal2html/blob/cc52c9b58b89049f7fdc226d4ba0159cad57aafb/signal2html/html.py#L171)). This could probably be rewritten to use the avatar colors too.

Reply:
Addressed in 03d11c0